interfaces/SVG.idl

```diff
@@ -692,3 +692,39 @@ partial interface SVGAElement {
 interface SVGViewElement : SVGElement {};
 
 SVGViewElement includes SVGFitToViewBox;
+
+
+////////////////////////////////////////////////////////////////////////////////
+// SVG 1.1 interfaces that do not exist in SVG 2 and that are still implemented.
```

Comment:
I don't think we should test these interfaces in WPT, in fact we should test that they're not supported.

@dontcallmedom how do you think we should handle this? I was tracking this as custom IDL in mdn-bcd-collector and that worked fine.

Reply:
I can imagine several approaches:
* they could be hardcoded as to be ignored here in WPT,
* they could be annotated with a custom extended attribute in webref (e.g. `[Deprecated]`) with WPT handling that extended attribute in a useful way,
* we could revisit their inclusion in webref in the first place - https://github.com/w3c/webref/issues/278 is where the discussion happened

Reply:
OK, I've commented on https://github.com/w3c/webref/issues/278 :)